source/develop/release-management/features/storage/cinderlib-integration.html.md

```diff
@@ -267,6 +267,32 @@ Cinderlib will be packaged in RPM.
 The Cinder drivers and Cinderlib needs to be installed in the Engine.
 OS-Brick needs to installed on the hosts
 
+#### Manual Install
+
+Since that some of the required libraries are currently not part of oVirt installation, the following steps are required:
+
+On Engine:
+
+```
+$ yum install -y centos-release-openstack-pike
+$ yum-config-manager --enable openstack-pike
+$ yum install -y openstack-cinder python-pip
+$ pip install cinderlib
+```
+
+On all the hosts: (Vdsm needs to be restarted after installing the packages)
+
+```
+$ yum install -y centos-release-openstack-pike
+$ yum install -y python2-os-brick
+```
+
+
+For Ceph backend, the following package is needed in the Engine and in all the hosts:
+
+```
+$ yum install ceph-common
+```
```

Comment:
What about Fedora? maybe packaging is in better shape on Fedora?

Reply:
Looks good , just 2 comments:

1) About installing 'ceph-common' on the hosts, do we need any restart (VDSM) afterwards? if so please add that restart is required.
2) Perhaps its best to change all 'install' to 'yum install -y'.